contracts/rewards/src/state.rs

```diff
@@ -26,7 +26,7 @@ pub struct StoredParams {
 pub struct EpochTally {
     pub contract: Addr,
     pub event_count: u64,
-    pub participation: HashMap<Addr, u64>,
+    pub participation: HashMap<String, u64>, // maps a worker address to participation count. Can't use Addr as key else deserialization will fail
```

Comment:
Why change this?

Reply:
Using Addr as a key in a HashMap will cause deserialization to fail. I don't really understand why, but it does. Without this, the tests crash.